ci/infra/azure/output.tf

```diff
@@ -0,0 +1,42 @@
+output "username" {
+  value = var.username
+}
+
+output "bastion_host" {
+  value = var.create_bastionhost ? azurerm_bastion_host.bastionhost.0.dns_name : "creation disabled"
+}
+
+output "ip_load_balancer" {
+  value = {
+    "public_ip" : azurerm_public_ip.lb.ip_address,
+    "fqdn" : azurerm_public_ip.lb.fqdn,
+  }
+}
+
+output "masters_public_ip" {
+  value = zipmap(
+    azurerm_linux_virtual_machine.master.*.name,
+    azurerm_linux_virtual_machine.master.*.public_ip_address,
+  )
+}
+
+output "masters_private_ip" {
+  value = zipmap(
+    azurerm_linux_virtual_machine.master.*.name,
+    azurerm_linux_virtual_machine.master.*.private_ip_address,
+  )
+}
+
+output "workers_public_ip" {
+  value = zipmap(
+    azurerm_linux_virtual_machine.worker.*.name,
+    azurerm_linux_virtual_machine.worker.*.public_ip_address,
+  )
+}
```

Comment:
Should be dropped if we do not expose worker nodes to the open

Reply:
just removed, enable the worker node public ip for automated deployment script only.